README.md

```diff
@@ -523,8 +524,8 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :bulb: [50 Shades of Go: Traps, Gotchas, and Common Mistakes for New Golang Devs](http://devs.cloudimmunity.com/gotchas-and-common-mistakes-in-go-golang/)
 - :video_camera: [Golang for begineers - YouTube playlist](https://www.youtube.com/playlist?list=PLRAV69dS1uWSR89FRQGZ6q9BR2b44Tr9N)
 - :video_camera: [Let's go with golang - YouTube playlist](https://www.youtube.com/playlist?list=PLRAV69dS1uWQGDQoBYMZWKjzuhCaOnBpa)
-
----
+📹 : Video Resource: [let's create a virtual OS for begineers- YouTube playlist](https://youtu.be/C81eCx1KSYg?list=PL-Jc9J83PIiGhwxgJGiYxGG7zChL1PmAx)
```

Comment:
Change the link to be the link of the playlist and rename the link to the title of the playlist.

Also, fix the emoji as mentioned in previous comments.